v2/app/src/main/java/com/google/firebase/example/makeitso/ui/shared/LoadingIndicator.kt

```diff
@@ -0,0 +1,31 @@
+package com.google.firebase.example.makeitso.ui.shared
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import com.google.firebase.example.makeitso.ui.theme.DarkBlue
+import com.google.firebase.example.makeitso.ui.theme.LightYellow
+
+@Composable
+fun LoadingIndicator() {
+    ConstraintLayout(modifier = Modifier.fillMaxSize()) {
```

Comment:
Out of curiosity: Why are we using ConstraintLayout here? It looks like the same could be achieved with a `Box`.

Reply:
This was a personal preference to be honest -  also `ConstraintLayout` gives us a lot of flexibility if we want to grow the Loading UI at some point

Reply:
I think we're adding too much complexity for something that _might_ happen in the future. Plus, a `Box` would be less code compared to `ConstraintLayout`.

Reply:
Yes, that's fair! Changed to `Box`.